### PR TITLE
grenades quickfixs

### LINKED ISSUE #44479: Grenade trigger issues
#### Some issues after extensive testing on local:

1. Infrared emitters can't be attached at all. 

2. Mousetrap doesn't work. With a mousetrap attached to the grenade it just stays a normal grenade (5 second timer). Doesn't matter if the mousetrap was armed when attached or not. 
 
~~3. Using health sensor (off) on wire will have the grenade prime normally (5 second timer) instead of toggling the health sensor's on/off state. Attaching a health sensor (on) will have it work, but it will still prime like a normal grenade of used in hand. I think using it in hand should toggle it on/off.~~ Intented. 

~~4. Using wirecutters to disarm primed grenades no longer works.~~ It ever working was apparently a bug. 

5. Disassembling an unsecured chemical grenade with a wrench will not make any attached trigger drop. Instead the trigger is still attached to the non-wired grenade casing. A QoL change would be to make them drop, so you can easily mass dissassemble grenades to use their triggers for something else (such as large grenades). 

~~6. We can probably remove most igniters from chemistry, since they're no longer used for triggers. 2 per chem lab for heating stuff would be enough.~~ Just bloats this issue. 
 
7. You can't remove a trigger from an unsecured grenade without using a multitool or another trigger on it first to open the hacking window. But using a multitool will also open the extra "please enter a new detonation time" window due to @nemvar 's change, which is a bit annoying. I suggest making it let you activate it in hand instead of having to use a multitool. 

I really wish I didn't have to spend 2.5+ hours testing, when I just wanted to do some quick wiki editing. 

#44258

@Tlaltecuhtli

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
fixes: #44479
changelog not needed

### CODE REVIEW

**Comment by Angustmeta** [comment]
I'd like a maintainer to clarify if `4. Using wirecutters to disarm primed grenades no longer works.` is to be considered intended or not.

**Comment by ExcessiveUseOfCobblestone** [comment]
I didn't even know that existed tbh, let's try to retain that functionality.

**Comment by Angustmeta** [comment]
Turns out disarming live nades was not intended functionality. Me or someone else will have to PR it.

**Comment by Tlaltecuhtli** [comment]
you just need to add an attackby wirecutter check to do active = false and prime() to check if (!active) return

File: code/modules/assembly/health.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	materials = list(MAT_METAL=800, MAT_GLASS=200)
 	attachable = TRUE
 
-	var/scanning = FALSE
+	var/scanning = TRUE
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/assembly/health.dm]: what does changing the default for this guy fix?

Comment (Tlaltecuhtli) [reply code/modules/assembly/health.dm]: priming in hand and by assembly its two separed things for all except proxy sensors which can be turned on by the priming as qol
these things start off and you have to open a dumb giant window with only a bottom to turn them on which you are gonna do anyway
what he reported isnt a bug but intended

Comment (ExcessiveUseOfCobblestone) [reply code/modules/assembly/health.dm]: kk

**Pull request merged**